querytyper.js

```diff
@@ -114,8 +114,10 @@ queryTyperConfig.rootDirs.forEach(rootDir => {
     // Write exports file
     const exportsFileBody =
       generatedQueryNames.length > 0
-        ? generatedQueryNames.map(queryName => `export { ${queryName}, Result as ${queryName}Result, Arguments as ${queryName}Args } from './${queryName}.query';`).join('\n') + '\n'
+        ? generatedQueryNames.map(queryName => `export { ${queryName} } from './${queryName}.query';`).join('\n') + '\n'
```

Comment:
This solves the `Cannot re-export a type when the --isolatedModules flag is provided` issue.